pubsub/gcp/gcp.go

```diff
@@ -142,9 +142,26 @@ func NewPublisher(ctx context.Context, cfg Config, opts ...option.ClientOption)
 	if err != nil {
 		return nil, err
 	}
-
+	t := c.Topic(cfg.Topic)
+	// Update PublishSettings from cfg.PublishSettings
+	// but never set thresholds to 0.
+	if cfg.PublishSettings.DelayThreshold > 0*time.Millisecond {
```

Comment:
`> 0*time.Millisecond` - i think you can change these two to `> 0`

Reply:
Confirmed and updated.